src/main/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionController.java

```diff
@@ -117,6 +118,20 @@ public ResponseEntity<String> buildDeletedArtefactSubscribers(@RequestBody Artef
             "Deleted artefact third party subscriber notification request has been accepted");
     }
 
+    @PutMapping("/configure-list-types")
+    @ApiOperation("Endpoint to update list type for existing subscription")
+    @ApiResponses({
+        @ApiResponse(code = 201, message = "Subscription successfully updated for user: {userId}"),
+        @ApiResponse(code = 400, message = "This subscription object has an invalid format. Please check again."),
+        @ApiResponse(code = 403, message = NOT_AUTHORIZED_MESSAGE)
+    })
+    public ResponseEntity<String> configureListTypesForSubscription(@RequestBody @Valid SubscriptionDto sub) {
```

Comment:
Rather than taking in the full subscription DTO object, could this endpoint just take in the user ID (in the path maybe) and the array of list types to set? Would save needing to construct the object in the frontend

Reply:
As discussed, I have made the changes.